src/views/TradingCompetition/components/CompetitionHeadingText.tsx

```diff
@@ -0,0 +1,69 @@
+import { Heading } from '@pancakeswap-libs/uikit'
+import styled from 'styled-components'
+
+const H1SizeStyles = (theme) => `
+font-size: 48px;
+line-height: 110%;
+white-space: nowrap;
+
+${theme.mediaQueries.sm} {
+  font-size: 64px;
+}
+`
+
+const H2SizeStyles = (theme) => `
+font-size: 32px;
+line-height: 110%;
+white-space: nowrap;
+
+${theme.mediaQueries.sm} {
+  font-size: 40px
+}
+`
+
+const sharedStyles = (props) => `
+color: ${props.textColor ? props.textColor : '#ffffff'};
+background:  ${props.background ? props.background : '-webkit-linear-gradient(#7645d9 0%, #452a7a 100%)'};
```

Comment:
Styled component add the vendor prefixes for you. `linear-gradient` is probably enough. Same below

Reply:
Ah yea you're right on this one. Weirdly some of them do need a `-webkit-` prefix to work (background-clip, text-fill-color, text-stroke)